build/prepare/CheckPrereqs.targets

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Import Project="UbuntuCoreclrAndCoreFxDependencies.props" />
+
+	<Target Name="CheckPrereqs"
+			DependsOnTargets="CheckUbuntuAptGetDependencies;
+												CheckCentOsYumDependencies;">
+
+		<Message Text="Using Stage0 version:" />
+		<Exec Command="$(DotnetStage0) --version" />
+	</Target>
+
+	<Target Name="CheckUbuntuAptGetDependencies"
+			Condition=" '$(Rid)' == 'ubuntu.14.04-x64' ">
+
+		<Exec Command="dpkg -s %(UbuntuCoreclrAndCoreFxDependencies)" />
+
+		<Exec Command="dpkg -s %(DebianPackageBuildDependencies)" />
+
+		<OnError ExecuteTargets="ShowAptGetUsage" />
```

Comment:
This will make the code messy in that we'd need to combine the itemgroups or to do two targets. If we decide this is becoming a pain we'll add a ToolTask that will handle the scenario for us.